openstack_controller/tests/common.py

```diff
@@ -446,6 +451,11 @@ def post(self, url, *args, **kwargs):
         path_and_args = parsed_url.path + "?" + parsed_url.query if parsed_url.query else parsed_url.path
         path_parts = path_and_args.split('/')
         nova_microversion_header = kwargs['headers'].get('X-OpenStack-Nova-API-Version')
+        ironic_microversion_header = kwargs['headers'].get('X-OpenStack-Ironic-API-Version')
```

Comment:
This code is repeated in `get` and `post` methods.
Do you think can we move this piece of code to a `_get_microversion_path` receiving headers list as argument and returning the path.
With that, we move this common code to only one place and if changes are needed in the future it is easier for us.
**DRY principle: Don't Repeat Yourself**

Reply:
Sure, makes sense!